docs/api-examples/hubspot.mdx

```diff
@@ -0,0 +1,103 @@
+# HubSpot
+
+In this example, we will show you how you can get started working with the [HubSpot](https://hubspot.com/) API to produce a working use case that lists the companies in a HubSpot CRM.
+
+:::info Before you start
+
+You'll need a HubSpot Private App Access Token. Find out how to set that up in the [HubSpot private apps documentation](https://developers.hubspot.com/docs/api/private-apps).
+
+:::
+
+<div class="count-steps-h3">
+
+### Install & Authorize Superface
+
+Start by insalling the Superface CLI. You can use [Homebrew](https://brew.sh/) for this.
+
+```shell
+brew install superfaceai/cli/superface
+```
+
+If you don't have a Superface account already, you can [sign up here](https://superface.ai/create-account). You can use your account to authenticate the CLI.
+
+```shell
+superface login
+```
+
+### Prepare the documentation
+
+HubSpot's APIs are available as [Open API Specifications](https://api.hubspot.com/api-catalog-public/v1/apis), for this use case we will use the [Companies OAS document](https://api.hubspot.com/api-catalog-public/v1/apis/crm/v3/objects/companies).
+
+```shell
+superface prepare https://api.hubspot.com/api-catalog-public/v1/apis/crm/v3/objects/companies "hubspot"
+```
+
+Once the documentation has been indexed, you can create a profile for the use case you want to achieve, which in this case is to get a list of companies in the CRM.
+
+```shell
+superface new hubspot "list companies"
+```
+
+A file called `companies-management.read-companies.profile` will be created in the `superface` folder as a result. This is a Comlink file that decribes the input and output expectations that the API expects to achieve this use case.
+
+### Turn the Comlink into code
+
+To turn the Comlink profile into runnable code, use the following command and include the name of the provider and a use case profile you want to work with.
+
+```shell
+superface map hubspot companies-management/read-companies
+```
+
+### Add your API key
+
+If you don't already have your HubSpot API key in your environment, you can add it to your `.env` file as `HUBSPOT_TOKEN=<your-notion-api-key>`.
```

Comment:
typo: Since HubSpot uses API key security scheme, the env has a different name

```suggestion
If you don't already have your HubSpot API key in your environment, you can add it to your `.env` file as `HUBSPOT_API_KEY=<your-hubspot-api-key>`.
```